### DISCUSSION TITLE
Center Frequencies of Scattering1D Filterbank. Connection with frequency bands of Phonocardiogram

### DISCUSSION THREAD

**Discussion by oyste** [kymatio/kymatio#1015] [Q&A] [answered]
I am using temporal scattering with Phonocardiogram signals for mental stress classification on a self collected dataset. I am wondering if I can draw a connection between underlying frequency bands covered in the original signal by the scattering transform and the center frequencies S.meta()['xi']. This is of interest to me in a feature importance perspective, to discuss relevant frequency bands connecting to mental stress. 

1. Is my intuition correct that one can multiply these center frequency values with the sampling frequency (fs) of the input as:
 `xi_analysed_from_pcg  = S.meta()['xi']*fs` 
to obtain the center of frequencies relating to the PCG data represented by each of the scattering coefficients respectively?
2. Also is then the frequential bandwidth "covered" by each coefficient equal to:
`sigma_analysed_from_pcg = S.meta()['sigma']*fs` ?
3. Is this frequency relationship possible to discuss for second order coefficients?

Any help would be greatly appreciated.

**Comment by changhongw** [comment, marked as answer]
Hi @oyste, thanks for your question! 

The way you calculate the center frequencies and frequential bandwidths are correct: `xi_analysed =  S.meta()['xi']*fs` and `sigma_analysed = S.meta()['sigma']*fs`. This relationship is applicable to second order coefficients. 

You may find this example useful: https://github.com/changhongw/examod/blob/main/4_local_relevance_maps.ipynb. It shows how we calculate (cells 4 and 5, specifically) and visualize the first and second order frequencies in Hz.

**Reply by oyste** [reply]
Thank you so much for the insights @changhongw! Great example.
